### PR TITLE
Adding Searching Algorithm with Linear Search

### PR DESCRIPTION
**Pull request by NishantWankhade**
Please accept the pull request for this Hacktober fest.
And please let me know if you wanna some improvements.
Thanks.. :}

### CODE REVIEW

File: Algorithms/Searching/readme.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Searching
+
+Searching is algorithm for finding a certain target element inside a container.
```
Comment (shhossain) [inline Algorithms/Searching/readme.md:3]: Add more info

Comment (NishantWankhade) [reply Algorithms/Searching/readme.md:3]: ok sir

**Review by shhossain — APPROVED** [review]
Approved. Merging.

**Comment by shhossain** [comment]
@NishantWankhade yes in I will change it or You will create a PR?

**Pull request merged**